### PR TITLE
Admin : Projects Panel convert lizmap plugin version from int to sortable string

### PR DESCRIPTION
**Pull request by nworr**
Convert lizmap plugin  Version to unified string sortable (0X.0Y.0Z) , datatables render as human readable  

From 
Screenshot 2023-02-01 at 12-24-18 Admin - Lizmap projects - Administration

Screenshot 2023-02-01 at 11-49-36 Admin - Lizmap projects - Administration

Funded by 3liz

### CODE REVIEW

File: lizmap/modules/admin/zones/project_list.zone.php [hunk 1]
```diff
@@ -251,4 +252,31 @@ private function getProjectInspection($inspectionDirectoryPath, $projectRelative
 
         return $inspectionData;
     }
+
+    /**
+     * Transform int formatted version (from 5 or 6 integer) to s   ortable string .
```
Comment (Gustry) [inline lizmap/modules/admin/zones/project_list.zone.php:257]:
```suggestion
     * Transform int formatted version (from 5 or 6 integer) to sortable string .
```

File: lizmap/modules/admin/zones/project_list.zone.php [hunk 2]
```diff
@@ -251,4 +252,31 @@ private function getProjectInspection($inspectionDirectoryPath, $projectRelative
...
+     *
+     * Transform "10102" into "01.01.02"
+     * Transform "050912" into "05.09.12"
+     *
+     * @param string $intVersion the lizmap Qgis plugin version (not always int !!)
```
Comment (Gustry) [inline lizmap/modules/admin/zones/project_list.zone.php:262]:
```suggestion
     * @param string $intVersion the lizmap QGIS plugin version (not always int !!)
```

**Review by Gustry — COMMENTED** [review]
From UX point of view, LGTM

**Review by rldhont — APPROVED** [review]

**Comment by rldhont** [comment]
@nworr you can rebase and merge commits before merging the PR.

**Comment by Gustry** [comment]
I don't think this PR has been backported @nworr FYI ;-)

**Pull request merged**